packages/edit-site/src/utils/constants.js

```diff
@@ -1,5 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// Navigation
+export const NAVIGATION_POST_TYPE = 'wp_navigation';
+
+// Templates.
+export const TEMPLATE_POST_TYPE = 'wp_template';
+export const TEMPLATE_PART_P0ST_TYPE = 'wp_template_part';
+export const TEMPLATE_CUSTOM_SOURCE = 'custom';
+
+// Patterns.
+export const PATTERN_POST_TYPE = 'wp_block';
+export const PATTERN_DEFAULT_CATEGORY = 'my-patterns';
+export const PATTERN_THEME_TYPE = 'pattern';
+export const PATTERN_CORE_SOURCES = [
+	'core',
+	'pattern-directory/core',
+	'pattern-directory/featured',
+	'pattern-directory/theme',
+];
+export const PATTERN_SYNC_STATUSES = {
```

Comment:
Renamed from "type" to "status" since I thought it might be confused with post types.

Reply:
Just for background the reason for `type` versus `status` was to avoid any confusion with the idea that a pattern was currently `unsynced` because the sync had not been run yet and `full` because the sync had been run and completed successfully, ie. sync status gives the idea that is something that changes as a sync process runs, whereas `type` indicates more of a static setting. But I don't have a strong opinion either way on this.

Reply:
Thanks for the context. Also happy to switch it back if that's the case. I was running under the assumption they were just labels for statuses, and didn't reflect the status of any entity.